### DISCUSSION TITLE
Vaultwarden without Docker on Synology NAS

### DISCUSSION THREAD

**Discussion by ColinTimBarndt** [dani-garcia/vaultwarden#2735] [Q&A]
Hello, I have a Synology DS220j NAS which does not support Docker. It has an ARMv8 64-bit processor and I would like to store my passwords on this device. Where do I start when I want to get Vaultwarden to work without Docker?

**Comment by Masgalor** [comment]
Hi @ColinTimBarndt ,
I can not help you with anything regarding your Synology device, but at first you will need to get your hands on a compatible binary anyway.

The best place to start is the wiki:
 - [Third party packages](https://github.com/dani-garcia/vaultwarden/wiki/Third-party-packages)
 - [Pre built binaries](https://github.com/dani-garcia/vaultwarden/wiki/Pre-built-binaries)
 - [Building binary](https://github.com/dani-garcia/vaultwarden/wiki/Building-binary)

If you can find a ready to install package for your distro/arch use it.
If there is no package, but vaultwarden offers a docker-image for your architecture go for that pre built binary.
If nothing else works, you need to build the binary yourself.
